sources/packages/web/src/views/institution/locations/active-applications/ActiveApplicationSummaryData.vue

```diff
@@ -0,0 +1,229 @@
+            {{ dateString(slotProps.data.studyEndPeriod) }}
+          </span>
+        </template></Column
+      >
+      <Column
+        headerStyle="width: 20%"
+        field="applicationNumber"
+        header="Application #"
+        :sortable="true"
+      ></Column>
+      <Column
+        field="applicationStatus"
+        header="Status"
+        headerStyle="width: 20%"
+      >
+        <template #body="slotProps">
+          <StatusChipActiveApplication
+            :status="slotProps.data.applicationStatus"
+          />
+        </template>
+      </Column>
+      <Column header="Action" headerStyle="width: 20%">
+        <template #body="slotProps">
+          <v-btn
+            v-if="
+              slotProps.data.applicationStatus === ApplicationStatus.available
+            "
+            class="primary-btn-background"
+            @click="goToViewApplication(slotProps.data.applicationId)"
+            >Report a change</v-btn
+          >
+          <v-btn
+            v-if="
+              slotProps.data.applicationStatus === ApplicationStatus.completed
+            "
+            class="primary-btn-background"
+            @click="
+              goToViewScholasticStanding(slotProps.data.scholasticStandingId)
+            "
+            >View</v-btn
+          >
+        </template>
+      </Column>
+    </DataTable>
+  </content-group>
+</template>
+
+<script lang="ts">
+import { onMounted, ref, watch, computed } from "vue";
+import { useRouter } from "vue-router";
+import { InstitutionRoutesConst } from "@/constants/routes/RouteConstants";
+import { InstitutionService } from "@/services/InstitutionService";
+import {
+  DEFAULT_PAGE_LIMIT,
+  PAGINATION_LIST,
+  DataTableSortOrder,
+  DEFAULT_PAGE_NUMBER,
+  PageAndSortEvent,
+  PaginatedResults,
+  ApplicationStatus,
+} from "@/types";
+import { ActiveApplicationSummaryAPIOutDTO } from "@/services/http/dto";
+import { useFormatters } from "@/composables";
+const DEFAULT_SORT_FIELD = "applicationNumber";
+import StatusChipActiveApplication from "@/components/generic/StatusChipActiveApplication.vue";
+
+export default {
+  components: {
+    StatusChipActiveApplication,
+  },
+  props: {
+    locationId: {
+      type: Number,
+      required: true,
+    },
+    locationName: {
+      type: String,
+      required: true,
+    },
+    applicationStatus: {
+      type: String,
+      required: true,
+    },
+  },
+
+  setup(props: any) {
+    const router = useRouter();
+    const page = ref(DEFAULT_PAGE_NUMBER);
+    const pageLimit = ref(DEFAULT_PAGE_LIMIT);
+    const sortField = ref(DEFAULT_SORT_FIELD);
+    const sortOrder = ref(DataTableSortOrder.ASC);
+    const searchCriteria = ref();
+    const { dateString } = useFormatters();
+    const applications = ref(
+      {} as PaginatedResults<ActiveApplicationSummaryAPIOutDTO>,
+    );
+    const rowsPerPageOptions = computed(() =>
+      applications.value.results?.length > 10 ? PAGINATION_LIST : undefined,
```

Comment:
Are we doing it in other places? Not sure if we need to have a cornern about it.

Reply:
no, I think our plan was not to use this.